README.md

```diff
@@ -14,9 +15,11 @@ Please visit [README](https://github.com/notaryproject/.github/blob/main/README.
 > The Notary Project documentation is available [here](https://notaryproject.dev/docs/).
 
 ## Table of Contents
-- [Documentation](#documentation)
-- [Code of Conduct](#code-of-conduct)
-- [License](#license)
+- [notation-core-go](#notation-core-go)
+  - [Table of Contents](#table-of-contents)
+  - [Documentation](#documentation)
+  - [Code of Conduct](#code-of-conduct)
+  - [License](#license)
```

Comment:
Looks like we don't need to change this section?

Reply:
fixed